f600233d1b154d507b782f84604bcb12/update/sys_script_include_35c37d36977611106fa8b4b3f153af94.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
+    <sys_script_include action="INSERT_OR_UPDATE">
+        <access>package_private</access>
+        <active>true</active>
+        <api_name>x_g_dis_atat.MultipartHelper</api_name>
+        <caller_access/>
+        <client_callable>false</client_callable>
+        <description>A class that helps with generating multipart requests for EDMS document uploads.</description>
+        <name>MultipartHelper</name>
+        <script><![CDATA[var MultipartHelper = Class.create();
+MultipartHelper.prototype = {
+  initialize: function() {
+  },
+  createBody: function(packageSysId, documentSysId, indexFields) {
+    var endpoint = gs.getProperty("glide.servlet.uri") + "api/x_g_dis_atat/edms/edms-multipart-attachment";
+    var indexFieldsString = JSON.stringify(indexFields);
+
+    var internalRequest = new sn_ws.RESTMessageV2();
+    internalRequest.setEndpoint(endpoint);
+    internalRequest.setHttpMethod("GET");
+    internalRequest.setRequestHeader("Accept", "multipart/form-data");
+    internalRequest.setBasicAuth(
+      gs.getProperty("x_g_dis_atat.edms.atatClient.username"),
```

Comment:
Do we need to create a service account for these?

Reply:
We can take off authentication for the endpoint, but I didn't think that would be good for security. If it is okay to remove authentication then we can remove it and there wouldn't be any need for a user. Unless there is another way to authenticate without an account?